test/appium/tests/critical/test_pairing_devices_sync.py

```diff
@@ -3,6 +3,7 @@
 from tests.base_test_case import MultipleSharedDeviceTestCase, create_shared_drivers
 from tests.users import transaction_senders, basic_user, ens_user, ens_user_message_sender
 from views.sign_in_view import SignInView
+from views.home_view import HomeView
```

Comment:
is ens_banner_close_button unavailable without importing HomeView?

Reply:
yes, it's a new button that I just added to HomeView, so we need to import this view to the current file to use this button I suppose

Reply:
> yes, it's a new button that I just added to HomeView, so we need to import this view to the current file to use this button I suppose

I supposed that if we use other buttons from HomeView then ens_banner_close_button should be available to...

Reply:
got it. fixed too